nixos/modules/services/web-servers/h2o/vhost-options.nix

```diff
@@ -57,7 +58,8 @@ in
                 config.services.h2o.defaultHTTPListenPort
               '';
               description = ''
-                Override the default HTTP port for this virtual host.
+                Override the default HTTP port for this virtual host. See
+                <https://h2o.examp1e.net/configure/base_directives.html#listen>.
```

Comment:
These docs read like this can also be an attrset, yet the type here only says port.

Reply:
Ok, then please extend the "See... ", part, and maybe highlight this is only the port again?